### DISCUSSION TITLE
Does Uno ListView support groups?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#18655] [General]
In Skia, it does not seem the case, but the help mentions item groups (sticky group headers) so I'm confused if it is a bug or not.

Skia
![image](https://github.com/user-attachments/assets/415ccfa2-d78e-4cd9-81f3-bb47d6d6faa3)

WinUI
![image](https://github.com/user-attachments/assets/fa7f1ec7-d3db-4c2f-b4fb-61316396ea79)

https://platform.uno/docs/articles/uno-development/listviewbase-internals.html

**Comment by jeromelaban** [comment]
ListView on Skia does not yet support grouping: https://github.com/unoplatform/uno/issues/234. You can implement a similar feature using `ItemsRepeater` though.
